File: java/com/epicdragonworld/gameserver/network/packets/sendable/CharacterCreationResult.java
package com.epicdragonworld.gameserver.network.packets.sendable;

import com.epicdragonworld.gameserver.network.SendablePacket;

/**
 * @author Pantelis Andrianakis
 */
public class CharacterCreationResult extends SendablePacket
{
	public CharacterCreationResult(int result)
	{
		// Send the data.
		writeShort(3); // Packet id.
		writeByte(result);
	}
}

File: EpicDragonWorldServer/Network/Packets/Sendable/CharacterCreationResult.cs
/**
 * Author: Pantelis Andrianakis
 * Date: November 7th 2018
 */
public class CharacterCreationResult : SendablePacket
{
    public CharacterCreationResult(int result)
    {
        // Send the data.
        WriteShort(3); // Packet id.
        WriteByte(result);
    }
}
